From 02a8a3b38cc34ab861b35a214956cb8c5b491752 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Thu, 5 Nov 2009 14:38:48 +0800
Subject: [PATCH 082/280] ENGR00118073 SAHARA: Fix compile error when enable
 armv7a

If enable armv7-a when using code sourcery gcc 4.3.3 toolchain,
imx-lib met compiling error in SAHARA driver

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 sahara2/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sahara2/Makefile b/sahara2/Makefile
index 9c2be69..04cf38a 100644
--- a/sahara2/Makefile
+++ b/sahara2/Makefile
@@ -15,7 +15,7 @@ TARGET_ARCH=-DCONFIG_ARCH_MX51
 endif
 
 # Add compilation checks
-CFLAGS += -g -Wall -W -Wmissing-prototypes -Wstrict-prototypes -Wdeclaration-after-statement
+CFLAGS += -g -Wall -W -Wmissing-prototypes -Wstrict-prototypes -Wdeclaration-after-statement -fPIC
 
 OBJS= fsl_shw_rand.o  fsl_shw_hash.o  fsl_shw_sym.o  fsl_shw_user.o         \
       fsl_shw_auth.o  fsl_shw_hmac.o  fsl_shw_wrap.o  fsl_shw_keystore.o um_adaptor.o  sf_util.o
-- 
1.8.0

